### PR TITLE
[stable/filebeat] Allow to add more secrets

### PR DESCRIPTION
**Pull request by pkalemba**
#### What this PR does / why we need it:
Allow to add an then mount secrets
For example certficates

#### Checklist
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/filebeat/templates/secret.yaml [hunk 1]
```diff
@@ -10,3 +10,9 @@ metadata:
 type: Opaque
 data:
   filebeat.yml: {{ toYaml (default .Values.config .Values.overrideConfig) | indent 4 | b64enc }}
+  {{- if .Values.extraSecrets }}
+  {{- range $key, $value := .Values.extraSecrets }}
+  {{ $key }}: {{ $value |b64enc }}
```
Comment (sstarcher) [inline stable/filebeat/templates/secret.yaml]: add a space here `| b64enc`

Comment (pkalemba) [reply stable/filebeat/templates/secret.yaml]: Fixed

**Pull request merged**